packages/babel-preset-latest/README.md

```diff
@@ -0,0 +1,35 @@
+# babel-preset-latest
+
+> Babel preset including es2015, 2016, 2017.
```

Comment:
Did we agree on latest? I still think it is very ambiguous and not clear on its own what it does.
We also probably want independent package versions before releasing this.